spec/helpers/null_logger.rb

```diff
@@ -0,0 +1,9 @@
+require 'logger' unless defined? Logger
+
+class NullLogger < Logger
```

Comment:
this can't / shouldn't depend on the rails logger (though i think it's fair to rely on the rails logger if it's defined), so there's [this method](https://github.com/lumoslabs/pester/blob/tru_initial_commit/lib/pester.rb#L83)*

*that said, it shouldn't try to require logger every time you run that method...